elements/pfe-button/test/pfe-button.spec.js

```diff
@@ -0,0 +1,120 @@
+import { expect, elementUpdated } from "@open-wc/testing/index-no-side-effects";
+import { spy } from "sinon";
+import { createFixture } from "../../../test/utils/create-fixture";
+import PfeButton from "../dist/pfe-button.js";
+
+const element = `
+  <pfe-button>
+    <button>Button</button>
+  </pfe-button>
+`;
+
+const badElement = `
+  <pfe-button>
+    <div>Bad button</div>
+  </pfe-button>
+`;
+
+const attributeElement = `
+  <pfe-button>
+    <button id="myBtn" disabled type="reset">Button</button>
+  </pfe-button>
+`;
+
+const denyAttributeElement = `
+  <pfe-button>
+    <button style="background: red;">Button</button>
+  </pfe-button>
+`;
```

Comment:
The React test runner freaks out over this inline styling attribute.  JSX assumes that you are passing a properties object to the styles attribute specifically.  Because, of course it does.  :)